pep_sphinx_extensions/pep_theme/static/style.css

```
@@ -225,14 +226,14 @@ div.table-wrapper {
 table {
     width: 100%;
     border-collapse: collapse;
-    border-top: 1px solid var(--colour-background-accent);
-    border-bottom: 1px solid var(--colour-background-accent);
+    border-top: 1px solid var(--colour-background-accent-light);
+    border-bottom: 1px solid var(--colour-background-accent-light);
 }
 table caption {
     margin: 1rem 0 .75rem;
 }
 table tbody tr:nth-of-type(odd) {
-    background-color: var(--colour-background-accent);
+    background-color: var(--colour-background-accent-light);
 }
```

```suggestion
}
table tbody tr:hover {
    background-color: var(--colour-background-accent-strong);
}
```

Add "highlight" on hover